src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -2023,7 +2023,7 @@ module TransactWriteItemsRequest =
         | _ -> None
 
 
-type TransactionBuilder(?metricsCollector: (RequestMetrics -> unit)) =
+type Transaction(?metricsCollector: (RequestMetrics -> unit)) =
```

Comment:
Should there be a convenience `.Transaction` factory method on TableContext? It should be more discoverable, downside would be it may be less obvious why the table needs to be supplied...